interface/forms/sdoh/info.txt

```diff
@@ -0,0 +1 @@
+Social Screening Tool
```

Comment:
could add a 2nd line `Social` to specify that it will be located under the `Social` Tab in the encounter or it will be loaded under `Miscellaneous` which could later be changed in Forms->Admin.

Reply:
The corresponding documentation goes through the option of changing this from Miscellaneous to a customized label in the Forms Administration gui, so figured would leave this as is (ie. let it default to Miscellaneous).
btw, made a wiki page which is linked from the Supplementary section (main page and 7.0.0 user manual) and the New features for 7.0.0 : https://www.open-emr.org/wiki/index.php/Social_Screening_Tool (super basic for now, but at least gets it up there along with the awesome pdf guide)